crowbar_framework/lib/crowbar/upgrade_timeouts.rb

```diff
@@ -31,6 +31,7 @@ def values
         chef_upgraded: @timeouts_config[:chef_upgraded] || 1200,
         router_migration: @timeouts_config[:router_migration] || 600,
         lbaas_evacuation: @timeouts_config[:lbaas_evacuation] || 600,
+        set_network_agents_state: @timeouts_config[:set_network_agents_state] || 120,
```

Comment:
Recent experiences says: start with bigger timeout so we do not have to raise it later... :-)